app/backend/resources/terms_of_service.md

```diff
@@ -45,14 +46,26 @@ We do not sell any user data and will only use or process it for purposes in lin
 
-We use Google Analytics for analytics purposes. We use Fastmail for our email hosting, Sentry for automatic bug reports, and GitHub for code hosting and bug reports.
+We use Google Analytics for analytics purposes. We use Google Workspace for our email hosting, Sentry for automatic bug reports, and GitHub for code hosting and bug reports.
 
 We may add additional service providers as needed, and will make an effort to update this document.
 
+
 # 7. Termination
 
-If you are in breach of these terms or any other policies in place, we may at our discretion suspend or terminate your account.
+If you are in breach of these terms or any other rules, guidelines, or policies in place, we may, at our discretion, suspend or terminate your account.
+
+
+# 8. Additional terms for Strong Verification
+
+This section contains additional terms for those members who complete Strong Verification. Strong Verification is an optional safety feature that allows you to verify your profile information by scanning a Biometric passport with a supported NFC enabled device.
+
+We use a trusted third-party application called [Iris ID](https://passportreader.app/) by Iris Development AB to read your passport if you choose to complete Strong Verification. The app processes your data and verifies it before passing it back to our server. As per their [Privacy policy](https://passportreader.app/privacy), they will delete your personal information within 24 hours. By completing Strong Verification, you agree to be bound by their [Privacy policy](https://passportreader.app/privacy) and [Terms & conditions](https://passportreader.app/terms).
+
+When you complete Strong Verification, the data retrieved from your passport is encrypted with an asymmetric keypair in such a way that the server can no longer access it. There is a mechanism for this data to be decrypted and retrieved manually, but the key is stored separately offline and requires human intervention to use. The following information is additionally stored on the server for continued verification purposes: country of nationality, sex, date of birth, passport expiry date, and the last three digits of the passport number.
+
+You can request for this Strong Verification data (except for the minimal anti-abuse data) to be deleted from the server at any time.
+
+The minimal anti-abuse data of nationality, passport expiry date, and the last three digits of the passport number is stored for all successful verifications and cannot be removed. This data is used for safety and moderation purposes in order to avoid banned users from recreating accounts. We believe this is an appropriate compromise between providing a safe platform for our users, while not storing sensitive personal information beyond what you wish to share with the platform and other users.
```

Comment:
```suggestion
The minimal anti-abuse data of nationality, passport expiry date, and the last three digits of the passport number is stored for all successful verifications and cannot be removed. This data is used for safety and moderation purposes, for example, to prevent banned users from creating new accounts. We believe this is an appropriate compromise between providing a safe platform for our users, while not storing sensitive personal information beyond what you wish to share with the platform and other users.
```